src/components/App/App.js

```diff
@@ -1,80 +1,32 @@
-import React, {useState, useEffect} from 'react';
-import ItemList from '../ItemList/ItemList';
-import InputItem from '../InputItem/InputItem';
-import Footer from '../Footer/Footer';
-import styles from './App.module.css';
-
-
-const App = () => {
-	const initialState = {
-		items: [
-			{
-				value: 'Покормить кота',
-				isDone: true,
-				id: 1
-			},
-			{
-				value: 'Сходить на работу',
-				isDone: false,
-				id: 2
-
-			},
-			{
-				value: 'Полить цветы',
-				isDone: true,
-				id: 3
-			}
-		],
-
-		count: 3,
-	};
-
-const [items, setItems] = useState(initialState.items);
-const [count, setCount] = useState(initialState.count);
-
-useEffect(() => {console.log('componentDidMount')}, []);
-useEffect(() => {console.log('componentDidUpdate')});	
+import React from 'react';
+import { BrowserRouter as Router, 
+	Route, 
+	Link, 
+} from 'react-router-dom';
+import Todo from '../Todo/Todo';
+import About from '../About/About';
+import Contacts from '../Contacts/Contacts';
+import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['MenuItem' is defined but never used.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)